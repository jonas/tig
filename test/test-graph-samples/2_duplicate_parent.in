commit A B C
    Commit A - Merge C into B
commit B C E
    Commit B - Merge E into C
commit C D
    Commit C after D
commit E D
    Commit E after D
commit D F
    Commit D after F
commit F G H
    Commit F - Merge H into G
commit G H I
    Commit G - Merge I into H
commit I H
    Commit I after H
commit H
    Commit H
