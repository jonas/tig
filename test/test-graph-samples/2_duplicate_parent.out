●─╮ Commit A - Merge C into B
●─│─╮ Commit B - Merge E into C
●─╯ │ Commit C after D
│ ●─╯ Commit E after D
●─╯ Commit D after F
●─╮ Commit F - Merge H into G
●─│─╮ Commit G - Merge I into H
│ │ ● Commit I after H
◎─┴─╯ Commit H
